README.md

```diff
@@ -105,16 +105,41 @@ The general idea of portals is summed up above: inset pre-rendering, activation,
 
 ### Privacy threat model and restrictions
 
+The Portals design is intended to comply with the [W3C Target Privacy Threat Model](https://w3cping.github.io/privacy-threat-model/). This section discusses the aspects of that threat model that are particularly relevant to Portals and how the design satisfies them.
+
+A portal can contain either a same-site or cross-site resource. Same-site portals don't present any privacy risks, but cross-site resources risk enabling [cross-site recognition](https://w3cping.github.io/privacy-threat-model/#model-cross-site-recognition) by creating a messaging channel across otherwise-partitioned domains.
```

Comment:
Interesting that this focuses on site over origin, but I guess it makes sense.

My understanding is that we're planning to block various channels for cross-_origin_ portals. This might deserve a sentence or two.

Reply:
> My understanding of the goal is that: an origin's owner can opt into information passing between origins, but the end-user has to opt into information passing between sites. Obviously we're not entirely there yet.

This is probably reasonable, although I'll note that for largely architectural/implementation reasons we're trying to get rid of `document.domain`. (And I guess we're trying to get rid of cookies too?) So there might be a future where the end-user has to opt into information passing between origins.

> Origins are more predictable than sites, so that could be a good reason to block cross-origin channels: to make it easier for developers to understand the restrictions. Is that why we're planning to block the cross-origin channels too, or is there an attack I've missed?

I think that's right. Plus the general program above of slowly trying to get rid of cross-origin same-site information sharing mechanisms means that we don't want to add any _new_ ones.